kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/launcher/TeamCityConsoleReporterTest.kt

```diff
@@ -24,6 +26,13 @@ import io.kotest.matchers.string.shouldHaveLineCount
 import io.kotest.matchers.string.shouldStartWith
 import kotlin.reflect.KClass
 
+private val osName = System.getProperty("os.name").toLowerCase()
+
+class EnabledIfNotWindows : EnabledCondition {
+   override fun enabled(specKlass: KClass<out Spec>): Boolean = osName.contains("windows")
+}
+
+@EnabledIf(EnabledIfNotWindows::class)
```

Comment:
Now that I think about this more, the leading and trailing `\r` might be due to the clue invocation, but I'm not sure. I'll come back to this one.